components/Manual.js

```diff
@@ -0,0 +1,136 @@
+import styled from "styled-components";
+import { manualData } from "@/lib/db";
+
+const StyledSection = styled.section`
+  width: 100%;
+  height: auto;
+  padding: 1rem;
+`;
+
+const StyledWelcome = styled.h1`
+  text-align: center;
+  line-height: 2rem;
+  font-weight: 600;
+`;
+
+const StyledText = styled.p`
+  margin-top: 2rem;
+  margin-bottom: 3rem;
+  &:last-child {
+    text-align: center;
+    font-size: 1.5rem;
+    margin-bottom: 0;
+  }
+`;
+
+const StyledDetails = styled.details`
+  border: 1px solid var(--button-background);
+  background: var(--section-background);
+  border-radius: 30px;
+  padding: 0.5rem;
+  margin-bottom: 1rem;
+  &[open] {
+    border: 0;
+  }
+`;
+
+const StyledSummary = styled.summary`
+  font-weight: bold;
+  width: 100%;
+  padding: 0.5rem;
+`;
+
+const StyledOl = styled.ol`
+  margin-top: 1rem;
+  padding: 0;
+  list-style: none;
+  counter-reset: item;
+`;
+
+const StyledListItem = styled.li`
+  counter-increment: item;
+  margin: 0;
+  padding: 2rem;
+  position: relative;
+  &:nth-child(even):before {
+    content: counter(item);
+    position: absolute;
+    right: 0;
+    top: -1rem;
+    margin-right: -20px;
+    border-radius: 50%;
+    padding: 0;
+    height: 3rem;
+    width: 3rem;
+    line-height: 3rem;
+    background-color: var(--button-background);
+    text-align: center;
+    color: var(--main-dark);
+    font-size: 1.5rem;
+  }
+  &:nth-child(odd):before {
+    content: counter(item);
+    position: absolute;
+    left: 0;
+    top: -1rem;
+    margin-left: -20px;
+    border-radius: 50%;
+    padding: 0;
+    height: 3rem;
+    width: 3rem;
+    line-height: 3rem;
+    background-color: var(--button-background);
+    text-align: center;
+    color: var(--main-dark);
+    font-size: 1.5rem;
+  }
+  &:nth-child(even) {
+    border-right: 2px dotted var(--button-background);
+    border-top-right-radius: 30px;
+    border-bottom-right-radius: 30px;
+    border-bottom: 2px dotted var(--button-background);
+    padding-right: 2rem;
+  }
+  &:nth-child(odd) {
+    border-left: 2px dotted var(--button-background);
+    border-top-left-radius: 30px;
+    border-bottom-left-radius: 30px;
+    border-bottom: 2px dotted var(--button-background);
+    padding-right: 2rem;
+  }
+`;
+
+export default function Manual() {
+  return (
+    <>
+      <StyledSection>
+        <StyledWelcome>Welcome to the What a feeling app!</StyledWelcome>
```

Comment:
I would suggest to highlight the app name here:
```suggestion
        <StyledWelcome>Welcome to the <b>What a Feeling</b> App!</StyledWelcome>
```